Tests/DeepLearningTests/LayerTests.swift

```diff
@@ -34,6 +34,22 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(round(output), expected)
     }
 
+    func testMaxPool2D() {
+        let layer = MaxPool2D<Float>(poolSize: (2, 2), strides: (1, 1), padding:.valid)
+        let input = Tensor(shape: [1, 2, 2, 1], scalars: (0..<4).map(Float.init))
+        let output = layer.inferring(from: input)
+        let expected = Tensor<Float>([[[[3]]]])
+        XCTAssertEqual(round(output), expected)
```

Comment:
Please use just `output` instead of `round(output)` when calling `XCTAssertEqual`, for precise equality testing. `round` greatly changes the value of tensors and is not suitable for use here.

Could you please remove other usages of `round` in this file as well?